.github/workflows/est-ds-realm-test.yml

```diff
@@ -393,17 +393,39 @@ jobs:
         run: |
```

Comment:
Could you explain the comment regarding "... and check disabled"?  What "check" was disabled?  And if the "check" was disabled, why expect BAD request?

Reply:
Currently, subject check is in EST module and in CA profile. In the previous check the request with different subject was reject from EST subsystem. The above step remove the check from EST subsystem so the subject check is done only by the CA.

In theory, the EST subsystem could support multiple CA/profiles so it should be OK to decide where the subject should be verified.

Reply:
Ok, the comment might want to explain it more clearly.  Also, to avoid confusion, the original check in EST should be disabled by default so perhaps tests involving checks in the est module could be disabled once this ticket is done.